conjureup/controllers/newcloud/gui.py

```diff
@@ -70,32 +70,22 @@ def finish(self, credentials=None, back=False):
 
         region = None
         if credentials is not None:
-            common.save_creds(app.current_cloud, credentials)
             cloud_type = juju.get_cloud_types_by_name()[app.current_cloud]
-            if cloud_type == 'maas':
-                # when bootstrapping a MAAS controller that hasn't been added
-                # as a saved cloud, we have to provide the API endpoint as
-                # the "region" when bootstrapping
-                region = credentials.fields()[0].value
-
-            # XXX: Oracle is handled special case until that cloud is
-            # solidified and makes it into Juju's default cloud listing
-            if cloud_type == 'oracle':
-                # need to create a custom cloud here with endpoint for oracle
-                endpoint = credentials.fields()[3].value
-                oracle_config = {
-                    'type': 'oracle',
-                            'description': 'Oracle Cloud',
-                            'auth-types': ['userpass'],
-                            'endpoint': endpoint,
-                            'regions': {
-                                'uscom-central-1': {}
-                            }
-                }
+            if cloud_type == 'maas' or cloud_type == 'oracle':
```

Comment:
I dont think this would be to hard to implement on this PR. I'll look at that soon